util/trace_replay.cc

```diff
@@ -140,6 +157,39 @@ Status Replayer::Replay() {
         db_->Get(roptions, cf_map_[cf_id], key, &value);
       }
       ops++;
+    } else if (trace.type == kTraceIteratorSeek) {
+      uint32_t cf_id = 0;
+      Slice key;
+      DecodeCFAndKey(trace.payload, &cf_id, &key);
+      if (cf_id > 0 && cf_map_.find(cf_id) == cf_map_.end()) {
+        return Status::Corruption("Invalid Column Family ID.");
+      }
+
+      if (cf_id == 0) {
+        single_iter = db_->NewIterator(roptions);
+      } else {
+        single_iter = db_->NewIterator(roptions, cf_map_[cf_id]);
```

Comment:
This is fine for now, but the right to handle this, in my opinion, is that we should trace the `NewIterator` operation too. Otherwise, we end up creating a new iterator for every Seek/SeekForPrev/Next/etc operation ... which would result in incorrect replay statistics.

Reply:
Yes. From the tracing-analyzing's perspective, recording Seek and SeekForPrev are able to know where the iterator starting point is. Even we have the "next" in the trace, for some analyzing case, if users do not import the whole key-space data, analyzer is unable to know what the Next is. 

But yes, from the trace_replay perspective, it is more complicated, we may need to track the: 1) NewIterator 2) ReFresh 3) Seek 4) SeekForPrev 5) SeekToFirst 6) SeekToLast 7) Next 8) Prev or even more